packages/cli-core/src/package-manager-controller/package_manager_controller_factory.ts

```diff
@@ -14,17 +13,13 @@ export class PackageManagerControllerFactory {
   /**
    * constructor
    * @param projectRoot - the root directory of the project
-   * @param userAgent - the user agent of the package manager
    */
-  constructor(
-    private readonly projectRoot: string,
-    private readonly userAgent: string
-  ) {}
+  constructor(private readonly projectRoot: string = './') {}
```

Comment:
another place where `projectRoot` should be injected and don't have default.

additionally.
This is the place `printer` should be injected from cli and create-amplify and flow to implementations.

Reply:
> another place where projectRoot should be injected and don't have default.

if we remove the default here, that means there's no default value for any `projectRoot`

Reply:
done [592cd72](https://github.com/aws-amplify/amplify-backend/pull/947/commits/592cd72fa98d3a5bff41cb2c303ced6bade38f1a)